reactioncommerce/reaction
# chore(catalyst): use new Catalyst Chip + fix sidebar colors

Impact: **minor**  
Type: **feature|bugfix|chore**

## Issue: Update Catalyst, material-ui/core and fix regressions
- Update Catalyst so we can use the new components, like the new deletable Chip types
- This Catalyst update introduced defaultTheme changes for the secondary color theme, which rendered the Sidebar much darker than usual. Add some one-off CSS for this one-off Sidebar style.

## Solution
- Upgrade Catalyst. Delete CSS that was used to make the buttons look like the actual spec (for the Action Dropdown)
- Add one-off styles with defaultTheme variables to cover the Sidebar. I did not add a new theme for the Sidebar b/c it's unclear whether we'll use this one-off type/color palette anywhere else

<img width="544" alt="Screen Shot 2019-08-21 at 5 01 54 PM" src="https://user-images.githubusercontent.com/3673236/63476554-2c2dfc00-c436-11e9-9e5a-3a5d549a5d9c.png">
<img width="309" alt="Screen Shot 2019-08-21 at 5 02 06 PM" src="https://user-images.githubusercontent.com/3673236/63476555-2c2dfc00-c436-11e9-8f61-f654c67f4385.png">
<img width="328" alt="Screen Shot 2019-08-21 at 5 02 02 PM" src="https://user-images.githubusercontent.com/3673236/63476556-2c2dfc00-c436-11e9-8e40-761edf998443.png">
<img width="1440" alt="Screen Shot 2019-08-21 at 5 02 17 PM" src="https://user-images.githubusercontent.com/3673236/63476557-2cc69280-c436-11e9-97fb-256a4ad630a7.png">
<img width="1440" alt="Screen Shot 2019-08-21 at 5 02 24 PM" src="https://user-images.githubusercontent.com/3673236/63476558-2cc69280-c436-11e9-9d84-fde83638d613.png">

- Update the Filter by File flow to use the new deletable Chip types:
<img width="1440" alt="Screen Shot 2019-08-21 at 6 03 08 PM" src="https://user-images.githubusercontent.com/3673236/63478285-13c1df80-c43e-11e9-9443-d77093ff35a3.png">
<img width="1440" alt="Screen Shot 2019-08-21 at 6 03 16 PM" src="https://user-images.githubusercontent.com/3673236/63478286-13c1df80-c43e-11e9-9fbe-d1773c3813f2.png">
<img width="1145" alt="Screen Shot 2019-08-21 at 6 03 11 PM" src="https://user-images.githubusercontent.com/3673236/63478287-13c1df80-c43e-11e9-8a89-084cea607cf0.png">
<img width="1166" alt="Screen Shot 2019-08-21 at 6 03 19 PM" src="https://user-images.githubusercontent.com/3673236/63478288-13c1df80-c43e-11e9-9453-bc3d27fe632f.png">


## Breaking changes
-None

## Testing
1. Make sure Sidebar looks good in Mobile and Desktop
1. Make sure Action dropdown has no style regressions.
